app/Models/User.php

```diff
@@ -379,6 +390,16 @@ public function getIsMutedAttribute(): bool
         return $this->isMuted();
     }
 
+    public function getIsUnrankedAttribute(): bool
+    {
+        return $this->isUnranked();
+    }
+
+    public function getIsBannedAttribute(): bool
+    {
+        return $this->isBanned();
+    }
```

Comment:
Should these be on ActsAsCommunityMember?

Reply:
Went ahead and moved them there in latest. I kept `isModerated()` in `User`, as I figured it's plausible some moderation action has happened on the user that isn't necessarily tied to the concerns of `ActsAsCommunityMember`.